user-guide/live-traffic/web-sessions/web-sessions-toolbar.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Web Sessions Toolbar
+slug: web-sessions-toolbar
+tags: web sessions toolbar, fiddler toolbar, share sessions
+publish: true
+position: 20
+---
+
+# Web Sessions Toolbar
```

Comment:
1) This article has a lot of images that "hide" some of the one liners of text. Consider reducing the amount of images by merging the info of 3-4 images in one and having more text in between to make it more visible.

Reply:
TODO